src/playlist/playlistmanager.cpp

```diff
@@ -237,6 +239,29 @@ void PlaylistManager::Save(const int id, const QString &filename, const Playlist
 
 }
 
+bool PlaylistManager::CreateAPlaylistsDirectory(const QString playlist_folder_path) {
+  bool success = QDir(playlist_folder_path).mkpath(playlist_folder_path);
+  if (! success) {
+    qLog(Debug) << "Could not create folder: " << playlist_folder_path;
+  }
+  return success;
+
+}
+
+void PlaylistManager::SaveAllPlaylists() {
+  for (const auto playlist : GetAllPlaylists()) {
+    QString playlist_folder_path = QDir::home().path() + QDir::separator() + "Playlists";
```

Comment:
I think we should show a select folder dialog instead of hard-code the path like this.